src/RoadRegistry.BackOffice.Api/ZipArchiveWriters/ForEditor/RoadSegmentsToZipArchiveWriter.cs

```diff
@@ -67,7 +67,7 @@ await using (var dbfEntryStream = dbfEntry.Open())
                     var cachedStreetNameIds = dbfRecords
                         .Select(record => record.LSTRNMID.Value)
                         .Union(dbfRecords.Select(record => record.RSTRNMID.Value))
-                        .Where(streetNameId => streetNameId.HasValue)
+                        .Where(streetNameId => streetNameId.HasValue && streetNameId.Value >= 0)
```

Comment:
`-9` records don't appear in StreetNameRegistry. Not sure what their name should be though ...